### ISSUE TITLE
Change Trader references to IDs

### ISSUE THREAD

**Issue by thaddeus** [TarkovTracker/tarkovdata#104]
Change the Giver and Reputation Trader references from English names to IDs

From:
```
        "giver": "Prapor",
        "turnin": "Prapor",
        "title": "Debut",
        "locales": {
            "en": "Debut",
            "ru": "Проба пера"
        },
        "wiki": "https://escapefromtarkov.gamepedia.com/Debut",
        "exp": 600,
        "unlocks": [
            "57dc2fa62459775949412633"
        ],
        "reputation": [
            {
                "trader": "Prapor",
                "rep": 0.08
            }
        ],
```
to
```
        "giver": 0,
        "turnin": 0,
        "title": "Debut",
        "locales": {
            "en": "Debut",
            "ru": "Проба пера"
        },
        "wiki": "https://escapefromtarkov.gamepedia.com/Debut",
        "exp": 600,
        "unlocks": [
            "57dc2fa62459775949412633"
        ],
        "reputation": [
            {
                "trader": 0,
                "rep": 0.08
            }
        ],
```

Also I think its safe to get rid of the `turnin` property. That only semi-applied to postman when I was creating the original schema.

**Comment by obj** [comment]
Also, we got 3 more properties to change. Require loyalty, reputation failure, and, as I remember, 2 quests with objectives.

>     {
>         "id": 8,
>         "require": {
>             "level": 10,
>             "quests": [
>                 6
>             ],
>             "loyalty": [
>                 {
>                     "trader": "Prapor",
>                     "stage": 2
>                 }
>             ]
>         },


>         "reputationFailure": [
>             {
>                 "trader": "Therapist",
>                 "rep": -0.04
>             }
>         ],


>         "objectives": [
>             {
>                 "type": "reputation",
>                 "target": "Prapor",
>                 "number": 4,
>                 "location": "Any",
>                 "id": 16
>             },

**Comment by thaddeus** [comment]
Resolved in #110
